packages/@o3r/core/schematics/module/templates/nx/project.json.template

```diff
@@ -48,6 +49,21 @@
           "<%= path %>/<%= name %>/src/**/*.ts"
         ]
       }
+    },
+     "prepare": {
+       "executor": "nx:run-script",
+       "outputs": [
+         "{projectRoot}/src/package.json"
```

Comment:
So that mean that this file should also be nxignored (or removed from `source` inputs of the other tasks)